### ISSUE TITLE
Disabled MaterialIcon changes size due to CSS styles

### ISSUE THREAD

**Issue by esoco** [GwtMaterialDesign/gwt-material#321]
The CSS styles for disabled MaterialIcons are inconsistent. They add a padding of 4px for disabled icons, making the layout "jumpy" on disabling. In addition they add a gray background while the enabled icon has no background at all so that the gray background seems inconsistent. This can be prevented by overriding the styles in the application CSS but it causes an inconvenient "out-of-the-box experience".

**Comment by BenDol** [comment]
Thanks for the report! Do you think you could put a small pull request together with the fix? Would be a great help! Otherwise we will fix it when we get a chance. Thanks!

**Comment by esoco** [comment]
Unfortunately I currently don't have a GwtMaterial project setup to test changes for side effects. The sizing problem seems to be caused by the following entry in `overridecss.css`, line 38:
```
i.disabled {
	padding: 4px;
}
```
This is a very generic term so I don't know what side effects a change might have. On the other hand I think that setting a sizing property only for disabled elements should be avoided. The gray background is set in the style before (line 29) for `button.disabled, i.disabled`. I would suggest to remove `i.disabled` from that style and it separately in the i.disabled block, but without the padding:
```
i.disabled {
	color: #9F9F9F !important;
}
```

**Comment by BenDol** [comment]
Agree'd not sure why its setting the padding in the first place. Good catch this will be patched for 1.5.1.
